source/php/Component/Hero/Hero.php

```diff
@@ -64,5 +64,28 @@ public function init()
         } else {
             $this->data['hasAnimation'] = false;
         }
+
+        /* Different hero types */
+        if($type) {
+            $this->data['contentSlotHasData'] = $this->slotHasData('content');
```

Comment:
Should this be wrapped in if-statement? It may be more clear if we always check this, eaven if it's not being used. Simplifies implementation if new functionality in the future.